druid/src/win_handler.rs

```diff
@@ -606,9 +606,7 @@ impl<T: Data> WinHandler for DruidHandler<T> {
         //it's possible that we should move those calls to occur in connected?
```

Comment:
we can also delete this comment while we're in here

Reply:
Whoops that comment is indeed outdated. I removed it.